frontend/src/components/filter/FilterRoles.js

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import FilterSelect from './FilterSelect';
+
+// when/if we use this status filter for a different model, we can pass these in as a prop instead
+// of defining these here
+
+const options = [
```

Comment:
Should these live in a db table?

Reply:
Or enum

Reply:
Since it's only used here, I don't think we need to create a table to store these values.